app/views/rooms/_nav.html.erb

```diff
@@ -0,0 +1,27 @@
+<nav class="not-zoomable">
+  <%= render 'shared/user_element' %>
+  <div class="mobile-content">
+    <% if @previous_room == nil %>
+        <i class="icon-left disabled"></i>
+    <% else %>
+        <a href="<%=
+            url_for controller: 'rooms', action: 'show', room_number:  @previous_room.id, week_number: @week.id
+
+        %>"><i class="icon-left"></i></a>
+    <% end %>
+    <% if @next_room == nil %>
```

Comment:
Lepsie je pouzit (to plati o vsetkych takychto porovnaniach):
`if @next_room.nil?`